integration_tests/src/main/python/string_test.py

```diff
@@ -939,6 +939,26 @@ def test_regexp_octal_digits():
             ),
         conf=_regexp_conf)
 
+def test_regexp_replace_digit():
+    gen = mk_str_gen('[a-z]{0,2}[0-9]{0,2}') \
+        .with_special_case('䤫畍킱곂⬡❽ࢅ獰᳌蛫青')
```

Comment:
@revans2 is it ok to add Unicode test data like this or should I convert it to `\unnnn` syntax?

Reply:
I think it is fine.